node/src/GlideClient.ts

```diff
@@ -589,51 +589,48 @@ export class GlideClient extends BaseClient {
      * Returns information about the function that's currently running and information about the
      * available execution engines.
      *
-     * See https://valkey.io/commands/function-stats/ for details.
+     * FUNCTION STATS runs on all nodes of the cluster, including primary and replicas.
```

Comment:
```suggestion
     * FUNCTION STATS runs on all nodes of the server, including primary and replicas.
```
That's on standalone installation